com.obsproject.Studio.Plugin.OBSLivesplitOne.yml

```diff
@@ -0,0 +1,39 @@
+id: com.obsproject.Studio.Plugin.OBSLivesplitOne
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//5.15-21.08
```

Comment:
I don't think that the right runtime

Reply:
```
> flatpak run com.obsproject.Studio
...
info: Distribution: "KDE Flatpak runtime" "5.15-21.08"
...
```
From what I can tell OBS seems to still be using that runtime

Reply:
nope

```
$ flatpak remote-info flathub com.obsproject.Studio
        ID: com.obsproject.Studio
       Ref: app/com.obsproject.Studio/x86_64/stable
      Arch: x86_64
    Branch: stable
Collection: org.flathub.Stable
  Download: 191.9 MB
 Installed: 501.7 MB
   Runtime: org.kde.Platform/x86_64/6.4
       Sdk: org.kde.Sdk/x86_64/6.4

    Commit: 817625e6baf9ff8e86467f3ebdcb2b65c47b8046bd5df9ea74afe794b60f4e48
    Parent: f503a59f4910533f9bcdfb88884782d6460e029c422a73d08cf12d3ee4d14144
   Subject: Export com.obsproject.Studio
      Date: 2023-02-04 12:32:29 +0000

```

Reply:
Huh, right
Guess this is just outdated then
```
[📦 org.kde.Sdk ~]$ cat /etc/os-release 
NAME="KDE Flatpak runtime"
VERSION="5.15-21.08"
VERSION_ID="5.15-21.08"
ID=org.kde.Platform
PRETTY_NAME="KDE Flatpak runtime"
BUG_REPORT_URL=https://bugs.kde.org/

```